packages/dapp-connector/README.md

```diff
@@ -5,5 +5,5 @@ persistence and set up web extension messaging.
 
 See [example web extension].
 
-[example web extension]: ../web-extension/e2e/extension
+[example web extension]: ../web-extension/
```

Comment:
`../web-extension` is a package, while this would be the actual example extension
```suggestion
[example web extension]: ../e2e/test/web-extension/
```